apps/products/serializers.py

```diff
@@ -1,9 +1,41 @@
 from rest_framework import serializers
 
-from .models import ProductCategory
+from ..brands.models import Brand
+from .models import ProductCategory, Products
 
 
 class ProductCategorySerializer(serializers.ModelSerializer):
     class Meta:
         model = ProductCategory
         fields = ["id", "name", "description"]
+
+
+class ProductSerializer(serializers.ModelSerializer):
+    brand = serializers.PrimaryKeyRelatedField(
+        queryset=Brand.objects.all(), write_only=True
```

Comment:
You can remove the write_only... we want this field to show when we read products

Reply:
fixed